src/ncp/spinel.c

```diff
@@ -2091,6 +2091,18 @@ const char *spinel_prop_key_to_cstr(spinel_prop_key_t prop_key)
         ret = "DEBUG_TEST_WATCHDOG";
         break;
 
+    case SPINEL_PROP_SERVER_ALLOW_LOCAL_DATA_CHANGE:
```

Comment:
Same for these guys, can you please move them after `SPINEL_PROP_SLAAC_ENABLED` to match the order from the header file (probably need a rebase on master as well to get the `SPINEL_PROP_SLAAC_ENABLED` definition).  Thanks.